src/components/viewsComponents/dashboard/utils/EventHelpers.test.ts

```diff
@@ -0,0 +1,105 @@
+import { IEvent } from 'components/viewsComponents/events/constants/EventsData';
```

Comment:
I would pref to have enter after settings data like : const pastEvents, ongoingEvent

Its not mistake its more like the pattern of code written. We like mixing it based of complex of variable. If its one line we dont make return before expect but if its more complex like here its nice to have enter